### ISSUE TITLE
Refine Definition of Reg_Info

### ISSUE THREAD

**Issue by OR13** [ietf-wg-scitt/draft-ietf-scitt-architecture#103]
```
Reg_Info = {
  ? "register_by": uint .within (~time),
  ? "sequence_no": uint,
  ? "issuance_ts": uint .within (~time),
  ? "no_replay": null,
  * tstr => any
}
```

https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/blob/0383ec2b5a23f11d1a42800aebbe38368e12376d/draft-ietf-scitt-architecture.md?plain=1#L670

IMO these are parameters on the issuer's view of the feed identifier.

sequence number is meaningless unless you know what topic you are trying to order.

**Comment by OR13** [comment]
I'd prefer to eliminate `Reg_Info` if possible, to keep the scitt envelopes as simple as possible, and to reuse related work, such as https://datatracker.ietf.org/doc/draft-ietf-cose-cwt-claims-in-headers/

Consider the following alternative to:

```

1 => alg // ES384

3 => content type 
// application/xml 
// application/json
// application/cbor

3 => kid 
// NzbLsXh8uDCcd-6MNwXF4W_7noWXFZAfHkxZsRGC9Xs  
// urn:ietf:params:oauth:jwk-thumbprint:sha-256:NzbLsXh8uDCcd-6MNwXF4W_7noWXFZAfHkxZsRGC9Xs
// 42
      
13 => CWT_Claims = {

  1 => iss, 
  // example.com
  // did:example:123
  // urn:uuid:9a652678-4616-475d-af12-aca21cfbe06d
  // fooBar

  2 => sub, 
  // pkg:docker/customer/dockerimage@sha256:244fd47e07d1004f0aed9c?repository_url=gcr.io
  // https://example.org/01/ { gtin } { ? exp }
  // 5bdd1802-b0d9-4c27-b86c-8df099ef377e
  // LibreOffice
  // liber_office

  3 => aud
  // transparency.vendor1.example
  // transparency.vendor2.example
  // transparency.regulator3.example
  
  5 => nbf
  // unix timestamp (not a secure timestamp)
  
  6 => iat
  // unix timestamp (not a secure timestamp)
  
  257 => [sueids](https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat-12#name-semi-permanent-ueids-sueids)
  
  
  ...
  
  393 => Reg_Info // all the scitt fields we need that are not already defined
  
```

**Comment by OR13** [comment]
"subject" is already used by many other standards:

- https://w3c.github.io/vc-data-model/#credential-subject
- https://pages.nist.gov/800-63-3/sp800-63-3.html

And probably others.... 

The dictionary definition of subject:

e
(1)
: the term of a logical proposition that denotes the entity of which something is affirmed or denied
also : the entity denoted
(2)
: a word or word group denoting that of which something is predicated

- https://www.merriam-webster.com/dictionary/subject

**Comment by OR13** [comment]
to furnish something essential to the development, sustenance, maintenance, or operation of...

b
: to supply (material to be operated on) to a machine
c
(1)
: to insert and deposit (something) repeatedly or continuously
(2)
: to insert and deposit something into (something)

- https://www.merriam-webster.com/dictionary/feed

A feed is what you put into or get out of a transparency service... feeds are "about subject", which are identified via "sub".

The input feed is the set of all signed statements accepted or rejected by a transparency service

The output feed is the set of all signed statements for which a receipt exists because a registration policy has been met.

This is what a feed looks like in factorio (input is iron ore and coal, output is steel plates):

![862972621_preview_Belt Smelter 3](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/assets/8295856/92ce71f2-7b7b-4180-987c-7f02097825b1)

The subject of each input might be "iron ore", "hematite", "https://www.ebi.ac.uk/chebi/searchId.do?chebiId=50818", or "chebi:50818"...

The subject of each output might be "steel plate 123", "ASTM709 Plate 456", or "urn:epc:tag:sgtin-96:1.7332402.026591.1234567890" ... 

CWT Claims can also be present in the protected headers of receipts.

**Comment by SteveLasker** [comment]
closing as resolved with #135
